larray/inout/excel.py

```diff
@@ -226,12 +228,15 @@ class PandasExcelHandler(FileHandler):
     """
     def __init__(self, fname, overwrite_file=False):
         super(PandasExcelHandler, self).__init__(fname, overwrite_file)
+        _, ext = os.path.splitext(fname)
+        self.read_engine = 'xlrd' if ext == '.xls' else 'openpyxl'
+        self.write_engine = 'xlwt' if ext == '.xls' else 'openpyxl'
```

Comment:
Doesn't pandas default to xlsxwriter for writing? At least it used to be the case (see https://stackoverflow.com/questions/51571026/engines-available-for-to-excel-function-in-pandas). We don't necessarily have to use the same default, but there might be a good reason for that (I don't really know). From the code below (_dump_item), it seems like we used it by default too (???), so do you have a good reason to change that?

Reply:
As I understand, 'xlsxwriter' does not support the old '.xls' format.
But OK, I can write:
```python
self.write_engine = 'xlwt' if ext == '.xls' else None
```

Reply:
I am unsure of the consequence of None, I just wondered why you did not use:
```python
self.write_engine = 'xlwt' if ext == '.xls' else 'xlsxwriter'
```

Reply:
Because `xlsxwriter` is an optional dependency. One may have installed `openpyxl` but not `xlsxwriter`. 
Or `xlsxwriter` is somehow not an optional dependency?

Reply:
well I consider it optional too.